tests/test_rds/test_rds.py

```diff
@@ -4,8 +4,10 @@
 import boto3
 import pytest
 from botocore.exceptions import ClientError
+from dateutil.tz import tzutc
+from freezegun import freeze_time
```

Comment:
Freezing time is a pain (doesn't work in server mode, etc.) and we should really only use it when absolutely necessary.  Isn't it enough to assert that `OrginalSnapshotCreateTIme == SnapshotCreateTime` or `SnapshotCreateTime > OriginalSnapshotCreateTime`?

Reply:
> `SnapshotCreateTime > OriginalSnapshotCreateTime`

The internal timekeeping of the DBSnapshot class uses only millisecond precision (just like AWS, so we shouldn't change that). But if the test code runs fast enough (<1ms), the assertion would fail or the test would become flaky. Right now the create+copy takes ~5ms on my system, but we never know what kind of optimizations people will come up with.

So I'd like to keep the freezing to ensure the test stays reliable.